stuffer/s2n_stuffer.c

```diff
@@ -152,6 +152,7 @@ int s2n_stuffer_wipe_n(struct s2n_stuffer *stuffer, const uint32_t size)
 
 int s2n_stuffer_release_if_empty(struct s2n_stuffer *stuffer)
```

Comment:
I think this is the wrong name for the function.

Reply:
I renamed the function to `s2n_stuffer_release_if_consumed`. Is that more appropriate?